src/components/connections/EditConnectionView.ts

```diff
@@ -58,15 +56,20 @@ export class EditConnectionView extends View {
           path.join(this.context.extensionPath, "templates", "editConn.js")
         )
       ),
-      isV1: conn.version === InfluxConnectionVersion.V1,
-      title: title,
-      connID: conn.id,
-      connName: conn.name,
-      defaultHostV1: defaultHostV1,
-      defaultHost: defaultHost,
-      connToken: conn.token,
-      connOrg: conn.org
```

Comment:
you also want to update `editConn.html` for these changes

Reply:
good catch